frontend/app/views/support/ABTest.scala

```diff
@@ -6,44 +6,77 @@ import java.net.URLEncoder
+      val filteredVariantList = filteredVariants.toList
+      val variantWeights = filteredVariantList.map(_.weight)
+      val weightSum = variantWeights.sum
+      val totalWeight = if (weightSum != 0.0) weightSum else 1
+      val cdf: Seq[Double] = variantWeights.scanLeft(0.0)(_ + _).tail.map(_ / totalWeight)
+      cdf.zip(filteredVariantList)
+    }
+
+    variantsByCountry.map { case (country, variants) => country -> addRanges(variants) }
+  }
 }
 
 object AmountHighlightTest extends TestTrait {
 
   def name = "AmountHighlightTest"
   def slug = "highlight"
   override type VariantFn = (CountryGroup, Option[Int]) => Html
-
+  private lazy val notAustralia: Set[CountryGroup] = Set(
+    UK,
+    US,
+    Canada,
+    NewZealand,
+    Europe,
+    RestOfTheWorld
+  )
   def variants = NonEmptyList(
-    Variant("Amount - 5 highlight","5",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(5))),
-    Variant("Amount - 25 highlight","25",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(25))),
-    Variant("Amount - no highlight","None",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100), None)),
-    Variant("Amount - 35 highlight","35",0,views.html.fragments.giraffe.contributeAmountButtons(List(10,35,65,100),Some(35))),
-    Variant("Amount - 35 highlight descending","35-descending",0,views.html.fragments.giraffe.contributeAmountButtons(List(100,65,35,10),Some(35))),
-    Variant("Amount - 100 highlight","100",0,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(100))),
-    Variant("Amount - 50 highlight","50",1,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(50))),
-    Variant("Amount - 15","15",0,views.html.fragments.giraffe.contributeAmountButtons(List(15,35,65,100),Some(35))),
-    Variant("Amount - 40 highlight","40",0,views.html.fragments.giraffe.contributeAmountButtons(List(20,40,75,100),Some(40)))
+    Variant("Amount - 5 highlight","5",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(5)),notAustralia),
+    Variant("Amount - 25 highlight","25",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100),Some(25)), notAustralia),
+    Variant("Amount - no highlight","None",0,views.html.fragments.giraffe.contributeAmountButtons(List(5,25,50,100), None), notAustralia),
+    Variant("Amount - 35 highlight","35",0,views.html.fragments.giraffe.contributeAmountButtons(List(10,35,65,100),Some(35)), notAustralia),
+    Variant("Amount - 35 highlight descending","35-descending",0,views.html.fragments.giraffe.contributeAmountButtons(List(100,65,35,10),Some(35)), notAustralia),
+    Variant("Amount - 100 highlight","100",0,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(100)), notAustralia),
+    Variant("Amount - 50 highlight","50",1,views.html.fragments.giraffe.contributeAmountButtons(List(25,50,100,250),Some(50)), notAustralia),
```

Comment:
This is the only pre existing variant with weight > 0 but just for clarity I added the "not australia filter to all of them"